packages/restapi/src/lib/pushapi/PushAPI.ts

```diff
@@ -156,10 +156,7 @@ export class PushAPI {
         ...options,
         version: options?.version || defaultOptions.version,
         versionMeta: options?.versionMeta || defaultOptions.versionMeta,
-        autoUpgrade:
-          options?.autoUpgrade !== undefined
-            ? options?.autoUpgrade
-            : defaultOptions.autoUpgrade,
+        autoUpgrade: true,
```

Comment:
This does not require any change.
decryptPGPKey default param need to be turned as on